tests/aws/services/secretsmanager/test_secretsmanager.snapshot.json

```diff
@@ -4586,5 +4582,339 @@
         }
       }
     }
+  },
+  "tests/aws/services/secretsmanager/test_secretsmanager.py::TestSecretsManager::test_rotate_secret_with_lambda_success[False]": {
+    "recorded-date": "15-03-2025, 08:34:38",
+    "recorded-content": {
+      "rotate_secret_immediately_1": {
+        "ARN": "arn:<partition>:secretsmanager:<region>:111111111111:secret:<SecretId-0idx><ArnPart-0idx>",
+        "Name": "<SecretId-0idx>",
+        "VersionId": "<version_uuid:2>",
+        "ResponseMetadata": {
+          "HTTPHeaders": {},
+          "HTTPStatusCode": 200
+        }
+      },
+      "describe_secret_rotated_1": {
+        "ARN": "arn:<partition>:secretsmanager:<region>:111111111111:secret:<SecretId-0idx><ArnPart-0idx>",
+        "CreatedDate": "datetime",
+        "Description": "testing rotation of secrets",
+        "LastChangedDate": "datetime",
+        "Name": "<SecretId-0idx>",
+        "NextRotationDate": "datetime",
+        "RotationEnabled": true,
+        "RotationLambdaARN": "<lambda-arn:1>",
+        "RotationRules": {
+          "AutomaticallyAfterDays": 1
+        },
+        "VersionIdsToStages": {
+          "<version_uuid:1>": [
+            "AWSCURRENT"
+          ]
+        },
+        "ResponseMetadata": {
+          "HTTPHeaders": {},
+          "HTTPStatusCode": 200
+        }
+      },
+      "list_secret_versions_rotated_1_1": {
+        "ARN": "arn:<partition>:secretsmanager:<region>:111111111111:secret:<SecretId-0idx><ArnPart-0idx>",
+        "Name": "<SecretId-0idx>",
+        "Versions": [
+          {
+            "CreatedDate": "datetime",
+            "KmsKeyIds": [
+              "DefaultEncryptionKey"
+            ],
+            "VersionId": "<version_uuid:1>",
+            "VersionStages": [
+              "AWSCURRENT"
+            ]
+          }
+        ],
+        "ResponseMetadata": {
+          "HTTPHeaders": {},
+          "HTTPStatusCode": 200
+        }
+      }
+    }
+  },
```

Comment:
nit: I don't think this is a valid test snapshot. I can't see where this function was run with the parameter `False` in the code

Reply:
you are right, I think I missed this while I was doing some refactoring and forgot to change it back again. It should not be False, it should be `None` indeed, I can update this